src/formatstringplus/widget/formatstringplus.js

```diff
@@ -28,7 +37,7 @@ dojo.declare('formatstringplus.widget.formatstringplus', mxui.widget._WidgetBase
     },

```

Comment:
`actLoaded` is deprecated can be removed

Reply:
Removed